tests/generate-site-redesign.test.js

```
@@ -58,6 +58,19 @@ test('script: filter wiring reads skill rows on input', () => {
   assert.match(index, /\[data-skill-row\]/);
   assert.match(index, /addEventListener\('input'/);
 });
+test('search: norm() folds separators so "review squad" finds review-squad', () => {
+  // The shipped script defines a one-line normalizer we can extract and execute.
+  const m = index.match(/const norm = (.*?);/);
+  assert.ok(m, 'interactive script defines a norm() helper');
+  const norm = eval('(' + m[1] + ')');
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)